src/VisualStudio/Core/Def/Implementation/Progression/ProgressionOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.Progression
+{
+    internal static class ProgressionOptions
```

Comment:
Does this need to be a non-static and implements `IOptionProvider`? What would be the effect?